### PR TITLE
Open up menus for data sidebars

### PR DESCRIPTION
**Pull request by janewang**

### CODE REVIEW

File: docs/data/hubble/data-catalog/data-model-diagram.mdx
Comment (ElliotFriend) [inline docs/data/hubble/data-catalog/data-model-diagram.mdx:6]:
```suggestion
:::info 

For more detailed information see the [Full Hubble Data Model Diagram](https://dbdiagram.io/d/Hubble-Public-Data-Model-66671e056bc9d447b1501801) or the [Compressed Hubble Data Model Diagram](https://dbdiagram.io/d/Hubble-Public-Data-Model-Compressed-666730996bc9d447b151a181).

:::
```
This might give some trouble with markdown parsing.

Comment (janewang) [reply docs/data/hubble/data-catalog/data-model-diagram.mdx]: Thanks, my vim auto formatted markdowns. Got look into that, updated

**Review by ElliotFriend — APPROVED** [review]
Looks great! Thanks!

**Comment by ElliotFriend** [comment]
@chowbao Does this get your approval? I'm not the expert when it comes to what is most useful to display to a user for Hubble, so I'll defer to your expertise

**Pull request merged**
